docs/web/using_with_svelte/installation.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: installation
+title: Install Tolgee Svelte SDK
+sidebar_label: Installation
+slug: /web/using_with_svelte/installation
+---
+
+
+:::info
+To get full image of working Svelte integration, check our [Svelte Example Application](https://github.com/tolgee/svelte-example).
+:::
+
+To install Tolgee Svelte integration library, run:
+
+
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: 'npm', value: 'npm', },
+    { label: 'yarn', value: 'yarn' },
+  ]
+  }>
+  <TabItem value="npm">
+
+```shell
+npm install @tolgee/svelte
+```
+
+  </TabItem>
+  <TabItem value="yarn">
+
+```shell
+yarn add @tolgee/svelte
+```
+
+  </TabItem>
+</Tabs>
+
+To install UI bundle for in-context translating you will also need to install package `@tolgee/ui`:
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: 'npm', value: 'npm' },
+    { label: 'yarn', value: 'yarn' },
+  ]
+  }>
+  <TabItem value="npm">
+
+```shell
+npm install @tolgee/ui
+```
+
+  </TabItem>
+  <TabItem value="yarn">
+
+```shell
+yarn add @tolgee/ui
+```
+
+  </TabItem>
+</Tabs>
+
+First, you will need to wrap your application code with `TolgeeProvider` component.
+
+```svelte
+<script lang="ts">
+  import { TolgeeProvider } from "@tolgee/svelte";
+  import UI from "@tolgee/ui";
+  import type { TolgeeConfig } from "@tolgee/core";
+
+  const tolgeeConfig = {
+    preloadFallback: true,
+    apiUrl: import.meta.env.VITE_TOLGEE_API_URL, // or provide your API URL directly
+    ui: import.meta.env.VITE_TOLGEE_API_KEY ? UI : undefined, // UI is not needed for production mode
+    apiKey: import.meta.env.VITE_TOLGEE_API_KEY, // or provide API_KEY directly
+  } as TolgeeConfig;
+</script>
+
+<TolgeeProvider config={tolgeeConfig}>
+  <div slot="loading-fallback">Loading...</div>
+  <slot />
+</TolgeeProvider>
+```
+
+If you bootstrapped your application with SvelteKit, your `.env.development.local` file should look like this:
+
+```shell
+VITE_TOLGEE_API_KEY=bkcq7tkoinjg9ehk9hc4t3duq3
```

Comment:
Maybe add link to integration, as a hint where to take the API key

Reply:
Created an issue, this cannot be done until there is no Tolgee Integration guide in server.